DOC-4390: Document the spaces cards list (#228)
exoplatform/exo-documentation: docs/Manage-Space.rst

=================

After logging in eXo Platform successfully, you will see a list of your

> last visited spaces with a short list on the left pane of the page, and you can see all spaces via **SPACES**.

Reviewer: last visited spaces with a short list on the left pane of the page, and you can see all spaces via **SPACES**
```suggestion
The 5 last visited spaces are visible on the left pane, and you can see all spaces via **SPACES** in the middle pane.
```

Author: Actually, we're displaying at least 5, if the user is member of more than 5 spaces, and we improved this behavior and make it related to the sidebar length, so i found that it will be too much developed to be described in the documentation, as long as the spaces number to display is different from a user to another.

|SP-1|

**Details**:

> -  |image1| The Filter box which allows you to quickly filter for a

Reviewer: ```suggestion
-  |image1| The Filter box which allows you to quickly look for a
```

   space from your spaces list.

-  |image2| The list of last visited spaces where you are a member or manager, classed by last visited first.

-  |image3| Allows you to access the Spaces page which displays all the
   spaces in the portal. Here, you can quickly join a space or search
   for your desired space.

> -  |image4| Allows you to quick create a new space.

Reviewer: ```suggestion
-  |image4| Allows you to quickly create a new space.
```

   
.. _note-last-visited-space:

 .. note::   You will see a short list of your last visited spaces just below the Last visited button.

.. _spaces-page:

The Spaces page includes the following sections:

|SP-2|

> - |image1| You will have many options on this section such as: Add a new space, information about the number of displayed space, 

Reviewer: ```suggestion
- |image1| This section contains the following options: Add a new space, information about the number of displayed space, 
```

   Filter spaces by name or description and a pulldown with choices All spaces and My spaces to filter the spaces list.  

 .. note:: All spaces, including your spaces, :ref:`visible
   spaces <About-space-visibility>`, and :ref:`hidden spaces <About-space-visibility>` where
   you are invited to become members, and my spaces for spaces where you are a member or manager.

> - |image2| Spaces Cards list in which you will have all spaces informations like,: Space name, avatar, cover 

Reviewer 2: ```suggestion
- |image2| Spaces' Cards list in which you will have all spaces information like Space name, avatar and cover 
```

Author: Spaces' or Space's ?

Reviewer 2: Spaces'

   and accessing button.    

>  .. note:: You can click on the information icon in the space card and then the card will flip and display the 

Reviewer 2: ```suggestion
 .. note:: You can click on the information icon of the space's card which will flip to display the 
```

>    space description and the list of managers.

Reviewer 2: ```suggestion
   space's description and its list of managers.
```

>    space description and list of managers.

Reviewer: ```suggestion
   space description and the list of managers.
```

   |SP-3| 

> - |image3| A block for spaces requests (invitations to join a space, pending requests sent, pending requests for your spaces and the list of your managing spaces.  

Reviewer: ```suggestion
- |image3| A block for spaces requests (invitations to join a space, pending requests sent, pending requests for your spaces and the list of the spaces your are managing.  
```

> - |image4| You will have a suggestion list that recommand some spaces that you can join. 
>    You can either send a request to join or decline the suggestion.   

Reviewer 2: ```suggestion
- |image4| A suggestion list of recommended spaces to join. 
   You can either send a request to join or decline the suggestion.   
```

=================

After logging in eXo Platform successfully, you will see a list of your
last visited spaces with a short list on the left pane of the page, and you can see all spaces via **SPACES**.

|SP-1|

**Details**:

-  |image1| The Filter box which allows you to quickly look for a
   space from your spaces list.

> -  |image2| The list of last visited spaces where you are a member or manager, classed by last visited first.

Reviewer 3: ```suggestion
-  |image2| The list of last visited spaces where you are a member or manager, sorted by last visit time.
```

> .. note::   The short list of your last visited spaces displays as many spaces as your screen allows it.

Reviewer 3: ```suggestion
.. note::   The short list of your last visited spaces displays as many spaces as your screen allows.
```

-  |image3| Allows you to access the Spaces page which displays all the
   spaces in the portal. Here, you can quickly join a space or search
   for your desired space.

> -  |image4| Allows you to quickly create a new space.

Reviewer 3: ```suggestion
-  |image4| Lets you quickly create a new space.
```

   
.. _note-last-visited-space:

>  .. note::   You will see a short list of your last visited spaces just below the Last visited button.

Reviewer 3: ```suggestion
 .. note::   You can see a short list of your last visited spaces just below the Last visited button.
```

.. _spaces-page:

The Spaces page includes the following sections:

|SP-2|

> - |image1| This section contains the following options: Add a new space, information about the number of displayed space, 

Reviewer 3: ```suggestion
- |image1| This section contains the following options: Add a new space, the number of spaces displayed, 
```

>    Filter spaces by name or description and a pulldown with choices All spaces and My spaces to filter the spaces list.  

Reviewer 3: ```suggestion
   Filter spaces by name or description and a pulldown to filter the list  : All spaces or My spaces.  
```

 .. note:: All spaces, including your spaces, :ref:`visible
   spaces <About-space-visibility>`, and :ref:`hidden spaces <About-space-visibility>` where
   you are invited to become members, and my spaces for spaces where you are a member or manager.

> - |image2| Spaces' Cards list in which you will have all spaces information like Space name, avatar and cover 

Reviewer 3: ```suggestion
- |image2| Spaces in which you can see information like space name, avatar and cover image
```

your collaborators can work together or discuss specific topics as
follows:

1. Click |image22| at the top left corner of the **Spaces** page to open the Add New Space form.

> 2. A drawer is displayed containing all information related to the space creation based on 3 steps: 

Reviewer 3: ```suggestion
2. A drawer panel is displayed containing all information related to the space creation based on 3 steps: 
```

   -  Close: The user cannot request for joining, but only the space
      manager can invite him.

.. note:: Visibility and registration fields have a description of the choice selected to help you choosing the best template for your needs when creating a new space.
      
.. note:: You can continue to the next step just by a click on **Continue**, and can also go back to the previous step by clicking on **Back**.

Step 3: Invite Users
~~~~~~~~~~~~~~~~~~~~~~~
.. _Users_invitation:      

> In the third, and last, step, the **Users** field, you can invite users or space members to the space to create just by picking them with the selector.

Reviewer: ```suggestion
In the third and last step, the **Users** field allows to pick with the selector the users or space members that you entend to invite ti the space you are creating.
```

   -  Close: The user cannot request for joining, but only the space
      manager can invite him.

.. note:: Visibility and registration fields have a description of the choice selected to help you choosing the best template for your needs when creating a new space.
      

> .. note:: You can continue to the next step just by a click on **Continue**, and can also go back to the previous step by clicking on **Back**.

Reviewer 3: ```suggestion
.. note:: You can continue to the next step with **Continue** button, and can also go back to the previous step with the **Back** button.
```

Step 3: Invite Users
~~~~~~~~~~~~~~~~~~~~~~~
.. _Users_invitation:      

> In the third and last step, the **Users** field allows to pick with the selector the users or space members that you entend to invite ti the space you are creating.

Reviewer 3: ```suggestion
In the third and last step, the **Users** field allows to pick users or space members that you intend to invite to the new space.
```

Type-ahead suggestions allow to facilitate selection users and spaces members to invite.

.. note:: Only spaces that the creator is member of can be selected. 

Users field can be pre-filled by default users or spaces defined in space template.

.. note:: You can go back to the previous step by clicking on **Back**.

|SP-9|

> Finally, you click on **Create** Space button to finish adding your new space. The new space appears.

Reviewer 3: ```suggestion
Finally, click **Create** button to finish adding your new space. The new space appears.
```

Only the space managers have permission to delete their spaces.

1. Open the Spaces pages, then select the space you want to delete.

> 2. Click **Remove** in the area button.

Reviewer: ```suggestion
2. Click on **Remove** in the area button.
```
